shared/chat/conversation/attachment-input/container.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import RenderAttachmentInput from './'
+import {connect} from 'react-redux'
+import {navigateUp} from '../../../actions/route-tree'
+
+import type {RouteProps} from '../../../route-tree/render-route'
+import type {TypedState} from '../../../constants/reducer'
+import type {AttachmentInput, SelectAttachment} from '../../../constants/chat'
+
+type AttachmentInputRouteProps = RouteProps<{
+  input: AttachmentInput,
+}, {}>
+type OwnProps = AttachmentInputRouteProps & {
+  title: string,
+}
+
+export default connect(
+  (state: TypedState, {routeProps, ...ownProps}: OwnProps) => {
+    const {input} = routeProps
+    return {
+      ...ownProps,
```

Comment:
if you're not going to reference ownProps here you can just not inject it since you're passing a merge function to connect. so instead below you can change ` (stateProps, dispatchProps)` to ` (stateProps, dispatchProps, ownProps)` and merge it there